src/main/pages/che-7/extensions/che4z-documentation-legal-notice.adoc

```diff
@@ -0,0 +1,16 @@
+=Documentation Legal Notice
+
+This Documentation, which includes embedded help systems and electronically distributed materials, (hereinafter referred to as the “Documentation”) is for your informational purposes only and is subject to change or withdrawal by CA at any time. This Documentation is proprietary information of CA and may not be copied, transferred, reproduced, disclosed, modified or duplicated, in whole or in part, without the prior written consent of CA.
```

Comment:
I'm fairly certain such conditions are not compatible with Eclipse Public License (https://www.eclipse.org/legal/epl-2.0/) under which the Eclipse Che docs website content is published.

I didn't realize Broadcom was intending to keep these conditions when publishing i Che upstream.

Is there any way you could make this content open source.